cmd/cli/main_test.go

```diff
@@ -3,13 +3,31 @@ package main
 import (
 	"errors"
 	"flag"
+	"os"
 	"testing"
 
 	"github.com/ing-bank/flink-deployer/cmd/cli/flink"
 	"github.com/stretchr/testify/assert"
 	"github.com/urfave/cli"
 )
 
+func TestGetAPITimeoutSecondsWithoutEnv(t *testing.T) {
```

Comment:
Maybe a bit more descriptive names for the 3 tests:

`TestGetAPITimeOutSecondsShouldReturnTheDefaultValueIfTheEnvVarIsUnset`
`TestGetAPITimeOutSecondsShouldReturnTheParsedValueFromTheEnvVar`
`TestGetAPITimeOutSecondsShouldReturnAnErrorIfTheEnvVarValueCannotBeParsed`